src/main/scala/com/geirsson/codegen/Codegen.scala

```diff
@@ -140,8 +140,31 @@ case class Codegen(options: CodegenOptions, namingStrategy: NamingStrategy) {
     s"""|package ${options.`package`}
         |${options.imports}
         |
+        |/**
+        | * Generated using [[https://github.com/olafurpg/scala-db-codegen scala-db-codegen]]
+        | *  - Number of tables: ${tables.size}
+        | *  - Database URL: ${options.url}
+        | *  - Database schema: ${options.schema}
+        | *  - Database user: ${options.user}
+        | */
         |//noinspection ScalaStyle
         |object Tables {
+        |
+        |  /**
+        |    * Quill used to have this trait before v1, but it's still useful to keep.
+        |    * Examples are: pattern matching on wrapped type and conversion to JSON objects.
+        |    */
+        |  trait WrappedValue[T] extends Any with WrappedType { self: AnyVal =>
```

Comment:
Since `WrappedValue` is static, could we move it somewhere outside the generated code?

- Easiest solution would probably be to put this into a separate source file and link to from the readme for people to copy-paste into their project.
- Another solution is put it into a separate project in the build and publish to Maven.